src/components/MultiSearch/index.tsx

```diff
@@ -545,6 +547,34 @@ export const MultiSearch: React.FC<MultiSearchProps> = ({
         searchTerms={searchValue.split(' ')}
         actionsCreator={actionsCreator}
         layerStyle={layerStyle}
+        onClick={(item: any) => {
```

Comment:
Is there a type for `item` available? If not, can we create one?

Reply:
No, there wasn't a type for `item` yet. It was also added in [50832de](https://github.com/terrestris/shogun-gis-client/pull/1399/commits/50832de8f122b0301791808cfcb479659376d980) and is now used there.